packages/components/pie-button/README.md

```diff
@@ -71,30 +67,6 @@ import { PieButton } from '@justeattakeaway/pie-button/dist/react';
 > When using `pie-button`, you will also need to include a couple of dependencies to ensure the component renders as expected. See [the PIE Wiki for more information and how to include these in your application](https://github.com/justeattakeaway/pie/wiki/Getting-started-with-PIE-Web-Components#expected-dependencies).
 
 
-## Local development
```

Comment:
I think we should still have the `Local Dev` and `Testing` headings and just have a sentence linking to the parts of the wiki that replace this info – so [this link](https://github.com/justeattakeaway/pie/wiki/Contributing-Guide#running-web-components) here, and the [testing section below](https://github.com/justeattakeaway/pie/wiki/Contributing-Guide#testing).

Reply:
resolved on slack 👍 have added the links under 'contributing' section:

Check out our [contributing guide](https://github.com/justeattakeaway/pie/wiki/Contributing-Guide) for more information on [local development](https://github.com/justeattakeaway/pie/wiki/Contributing-Guide#local-development) and how to run specific [component tests](https://github.com/justeattakeaway/pie/wiki/Contributing-Guide#testing).